updates/next.json

```diff
@@ -1,11 +1,11 @@
 {
   "stream": "next",
   "metadata": {
-    "last-modified": "2020-06-03T15:08:25Z"
+    "last-modified": "2020-06-03T16:12:25Z"
   },
   "releases": [
     {
-      "version": "32.20200601.1.1",
+      "version": "32.20200601.1.0",
```

Comment:
but `32.20200601.1.1` is right, isn't it? we respun the build in https://github.com/coreos/fedora-coreos-streams/issues/109#issuecomment-637919696

Reply:
The release index only has `.0` though.